sc2reader/resources.py

```diff
@@ -579,6 +579,7 @@ def register_default_readers(self):
         self.register_reader('replay.game.events', readers.GameEventsReader_26490(), lambda r: 26490 <= r.base_build < 27950)
         self.register_reader('replay.game.events', readers.GameEventsReader_27950(), lambda r: 27950 <= r.base_build < 34784)
         self.register_reader('replay.game.events', readers.GameEventsReader_34784(), lambda r: 34784 <= r.base_build)
```

Comment:
You need to put an upper bound on this condition now. Same as the ones before it.